indigo_app/static/javascript/indigo/utils.js

```diff
@@ -15,3 +15,21 @@ $(function() {
 
   $('body').on('click', 'a[data-confirm], button[data-confirm]', handleConfirm);
 });
+
+// Show popover when hovering on selected links
+$('body').on('mouseenter', 'a[data-popup-url]', function() {
```

Comment:
Will this allow us to click on links in the popup?

Reply:
Yes, you can still click on a link in the popup

Reply:
Great.